src/ORM/Marshaller.php

```diff
@@ -105,6 +105,18 @@ public function one(array $data, array $options = [])
         $propertyMap = $this->_buildPropertyMap($options);
 
         $schema = $this->_table->schema();
+        $primaryKey = $schema->primaryKey();
+
+        if (array_intersect($primaryKey, array_keys($data)) == $primaryKey) {
+            $record = $this->_table->find('all');
+            foreach ($primaryKey as $pkey) {
+                $record->where(["$pkey" => $data[$pkey] ]);
```

Comment:
Hi Lorenzo. Thanks for your comment. Honestly I don't see the possibility of collisions once I am not joining tables. But if you still think it is necessary to add table names I can do it without problem, I just kindly ask you to confirm.